tests/labhub_test.py

```diff
@@ -153,21 +159,24 @@ def test_create_issue_cmd(self):
         labhub.REPOS = {'repository': self.mock_repo,
                         'repository.github.io': self.mock_repo}
 
-        testbot_public.assertCommand('!new issue repository this is the title\nbo\ndy',
-                              'Here you go')
+        testbot_public.assertCommand('!new issue repository this is the '
```

Comment:
start the command string on the next line so that it can use the full width, and have line breaks in sensible spots.